### PR TITLE
Nyala 23/lectures navbar

### PR DESCRIPTION
**Pull request by kosenchiha**
mob
full-screen

### CODE REVIEW

File: src/templates/LectureView/index.js [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
   const { strapiLecture, strapiCourse } = data
-  const { url } = strapiLecture
+  const { url, title, position } = strapiLecture
+  const { title: courseTitle } = strapiCourse
+
+  const findLectureByPosition = index => {
+    return strapiCourse.lectures.find(
+      lecture => lecture.position === position + index
+    ).id
+  }
```
Comment (kabaros) [inline src/templates/LectureView/index.js:21]: can you make this defensive, by checking that the result of find is not null, before trying to do `.id`

File: src/templates/LectureView/index.js [hunk 2]
```diff
@@ -23,6 +31,55 @@ const LectureView = ({ data }) => {
...
+              <div className="flex flex-col lg:flex-row ml-auto">
+                <div className="px-3 py-2  text-xs font-bold leading-snug text-white ">
+                  {title}
+                </div>
+                <div className="flex">
+                  {position === strapiCourse.lectures.length ? (
+                    <div className="px-3 py-1 text-xl text-white text-gray-600 leading-snug">
+                      <AiFillForward />
+                    </div>
+                  ) : (
```
Comment (kabaros) [inline src/templates/LectureView/index.js:53]: I think this is repeated twice, so maybe put it in its small component

File: src/templates/LectureView/index.js [hunk 3]
```diff
@@ -23,6 +31,55 @@ const LectureView = ({ data }) => {
...
+                {courseTitle}
+              </Link>
+            </div>
+            <div className="lg:flex flex-grow items-center">
+              <div className="flex flex-col lg:flex-row ml-auto">
+                <div className="px-3 py-2  text-xs font-bold leading-snug text-white ">
+                  {title}
+                </div>
+                <div className="flex">
+                  {position === strapiCourse.lectures.length ? (
```
Comment (kabaros) [inline src/templates/LectureView/index.js:49]: maybe put `position === strapiCourse.lectures.length` in a variable called `isLastLecture` or something, to make the code easier to understand .. same for `position===1`

File: src/templates/LectureView/index.js (hunk 1 above)
Comment (kosenchiha) [reply src/templates/LectureView/index.js:21]: done

File: src/templates/LectureView/index.js (hunk 2 above)
Comment (kosenchiha) [reply src/templates/LectureView/index.js]: I extracted all logic for lecture navigation button in separate component.

File: src/templates/LectureView/index.js (hunk 3 above)
Comment (kosenchiha) [reply src/templates/LectureView/index.js]: done

**Pull request merged**